scripts/sct_maths.py

```diff
@@ -515,16 +523,34 @@ def compute_similarity(data1, data2, fname_out='', metric='', verbose=1):
     :param metric: 'mi' for mutual information or 'corr' for pearson correlation coefficient
     :return: None
     '''
+    import matplotlib
```

Comment:
It would be best practice to isolate the visualization to make integration to other tools easier.